### ISSUE TITLE
how  does one download test jars in ammonite?

### ISSUE THREAD

**Issue by bblfish** [VirtusLab/scala-cli#1418]
```scala
╭─hjs@fish.fritz.box ~/Scala3/CoSy/SolidCtrlApp  ‹CommandLine*›
╰─➤  scala-cli repl --amm
Scala 3.2.0 is not yet supported with this version of ammonite
Defaulting to Scala 3.1.3

Loading...
Welcome to the Ammonite Repl 2.5.4-22-4a9e6989 (Scala 3.1.3 Java 19)
@ interp.repositories() ++= Seq(coursierapi.MavenRepository.of("https://oss.sonatype.org/content/repositories/snapshots"))

@ import  $ivy.`net.bblfish.crypto::bobcats:0.2-69106e6-SNAPSHOT`
https://oss.sonatype.org/content/repositories/snapshots/net/bblfish/crypto/bobcats_3/0.2-69106e6-SNAPSHOT/bobcats_3-0.2-69106e6-SNAPSH…
  No new update since 2022-02-09 16:37:40
https://oss.sonatype.org/content/repositories/snapshots/net/bblfish/crypto/bobcats_3/0.2-69106e6-SNAPSHOT/bobcats_3-0.2-69106e6-SNAPSH…
  No new update since 2022-02-09 16:37:39
https://oss.sonatype.org/content/repositories/snapshots/net/bblfish/crypto/bobcats_3/0.2-69106e6-SNAPSHOT/bobcats_3-0.2-69106e6-SNAPSH…
  No new update since 2022-02-09 16:37:42
import  $ivy.$
```
I get that far, but then I would also like to import the test jar `bobcats_3-0.2-69106e6-SNAPSHOT-tests.jar` [in the same sonatype repository](https://oss.sonatype.org/content/repositories/snapshots/net/bblfish/crypto/bobcats_3/0.2-69106e6-SNAPSHOT/).  How do I do that?

**Comment by Gedochao** [comment]
So first things first, please don't ask questions by creating issues, they are not meant for that.
[Discord](https://discord.gg/KzQdYkZZza) would be the better place for it, or [a GitHub discussion](https://github.com/VirtusLab/scala-cli/discussions).

Secondly, I believe this is a question about the insides of [`Ammonite`](https://github.com/com-lihaoyi/Ammonite) (well, `Ammonite` and `coursier`), and not `scala-cli`. If this should be improved (or documented better) in `Ammonite`, please raise an issue in [the other repository](https://github.com/com-lihaoyi/Ammonite).

Now back to your question, I believe this will work:
```scala
interp.load.ivy( 
      "net.bblfish.crypto" %% "bobcats" % "0.2-69106e6-SNAPSHOT", 
      Dependency.of("net.bblfish.crypto", "bobcats_3", "0.2-69106e6-SNAPSHOT").withClassifier("tests")
) 
```

stuff from the test-jar seems to be on the classpath, so it'll probably be enough for you.

```scala
▶ scala-cli --amm
Loading...
Welcome to the Ammonite Repl 2.5.4-33-0af04a5b (Scala 3.2.0 Java 17.0.2)
@ import coursierapi._ 
import coursierapi._

@ interp.repositories() ++= Seq(coursierapi.MavenRepository.of("https://oss.sonatype.org/content/repositories/snapshots")) 


@ interp.load.ivy( 
      "net.bblfish.crypto" %% "bobcats" % "0.2-69106e6-SNAPSHOT", 
      Dependency.of("net.bblfish.crypto", "bobcats_3", "0.2-69106e6-SNAPSHOT").withClassifier("tests")
) 


@ bobcats.HashSuite 
-- Error: core/shared/src/test/scala/bobcats/HashSuite.scala:27:24 -------------
undefined: new munit.CatsEffectSuite # -1: TermRef(TypeRef(TermRef(ThisType(TypeRef(NoPrefix,module class <root>)),object munit),CatsEffectSuite),<init>) at parser
Compilation Failed

@  
```

AFAIK the following syntax should also be supported by `coursier`, but it somehow doesn't work in the `ammonite` `repl`.
```scala
import  $ivy.`net.bblfish.crypto::bobcats:0.2-69106e6-SNAPSHOT:test`
```
cc @alexarchambault

**Comment by bblfish** [comment]
Thanks @Gedochao, I had tried on [Li Haoyi's discord channel](https://discord.com/channels/632150470000902164/940067748103487558/1026404473096118302), but got no response. (perhaps my question was not well formed) 

It' no longer clear where to send the bug or questions anymore. It would help if your doc mentioned the `scala-cli repl --amm` option, and at that place what the responsibilities are, where to file bugs, etc... It would also help if https://ammonite.io/ mentioned the `scala-cli` implementation in its docs (I found that out by asking a question on Discord).  Btw. Does scala-cli have it's own equivalent to ammonite? 

Btw, from my experience of github repos, the adminstrator can always turn a bug report into a question. 

Let me follow your suggestions before I post a bug on the ammonite gihtub repo linking back to the discussion here.
